workflows/build.gradle

```diff
@@ -47,8 +48,9 @@ dependencies {
 
     // This are shared so should be here.
     // Dependencies Required By Test Tooling
-    testImplementation "net.corda:corda-simulator-api:$simulatorVersion"
-    testRuntimeOnly "net.corda:corda-simulator-runtime:$simulatorVersion"
+    // Todo:  these are commented out as the simulator UTXO work has not been merged into Gecko yet.
```

Comment:
cleanup dead comments

Reply:
We need to leave those lines in for when we add Simulator back in for testing simulator. They would be cleaned out before a release branch was cut.